source/Jobbr.Server/JobbrServer.cs

```diff
@@ -228,31 +226,31 @@ private void WaitForDb()
             {
                 try
                 {
-                    this.jobStorageProvider.IsAvailable();
+                    _jobStorageProvider.IsAvailable();
                     return;
                 }
                 catch (Exception ex)
                 {
-                    Logger.ErrorException("Exception while connecting to database to get all Jobs on startup.", ex);
+                    _logger.LogError("Exception while connecting to database to get all Jobs on startup: '{e}'", ex);
                     Thread.Sleep(1000);
                 }
             }
         }
 
         private void RegisterJobsFromRepository()
         {
-            Logger.Info("Adding jobs from the registry");
+            _logger.LogInformation("Adding jobs from the registry");
 
             try
             {
-                var numberOfChanges = this.registryBuilder.Apply(this.jobStorageProvider);
-                var numberOfJobs = this.jobStorageProvider.GetJobs(pageSize: int.MaxValue).Items.Count;
+                var numberOfChanges = _registryBuilder.Apply(_jobStorageProvider);
+                var numberOfJobs = _jobStorageProvider.GetJobs(pageSize: int.MaxValue).Items.Count;
 
-                Logger.InfoFormat("There were {0} changes by the JobRegistry. There are now {1} known jobs right available.", numberOfChanges, numberOfJobs);
+                _logger.LogInformation("There were {changeCount} changes by the JobRegistry. There are now {jobCount} known jobs right available.", numberOfChanges, numberOfJobs);
             }
             catch (Exception e)
             {
-                Logger.FatalException("Cannot register Jobs on startup. See Exception for details", e);
+                _logger.LogCritical("Cannot register Jobs on startup. See Exception for details: {e}", e);
```

Comment:
Pass the whole exception to the log: `_logger.LogCritical(e, "message", values)`